sdk/eventhub/azure-eventhubs/azure/eventhub/common.py

```diff
@@ -67,32 +67,22 @@ def __init__(self, body=None, **kwargs):
         :type batch: Generator
         :param to_device: An IoT device to route to.
         :type to_device: str
-        :param message: The received message.
-        :type message: ~uamqp.message.Message
         """
-        to_device = kwargs.get("to_device", None)
-        message = kwargs.get("message", None)
 
         self._partition_key = types.AMQPSymbol(EventData.PROP_PARTITION_KEY)
         self._annotations = {}
         self._app_properties = {}
         self.msg_properties = MessageProperties()
```

Comment:
Why do we duplicate the properties between the inner/nested message and this instance? This is a public attribute, and if I add/remove properties after `self.message` has been created, then they presumably get out of sync...